src/Drivers/Decorator.php

```diff
@@ -418,6 +418,17 @@ public function purge($features = null): void
         }
     }
 
+    /**
+     * Retrieve the feature's name.
+     *
+     * @param  string  $feature
+     * @return string
+     */
+    public function name($feature)
+    {
+        return $this->resolveFeature($feature);
+    }
+
```

Comment:
As the `resolveFeature` function is protected, I have introduced a new method for this so we didn't have to introduce a breaking change by changing the visibility.

Also, `name` is a much nicer public facing method name.